west.yml

```diff
@@ -0,0 +1,16 @@
+# Copyright (c) 2021 Nordic Semiconductor ASA
+# SPDX-License-Identifier: Apache-2.0
+
+manifest:
+  self:
+    path: example-application
+
+  remotes:
+    - name: zephyrproject-rtos
+      url-base: https://github.com/zephyrproject-rtos
```

Comment:
In the README the `west` command uses `git@...`, where this uses `https://`.  If you intend on showing both usages in the example, then add comments to each spot referencing alternatives to avoid confusion.

Reply:
done